monitoring/prometheus/rules/ecommerce-alerts.yml

```
@@ -0,0 +1,168 @@
+          summary: "Kong Gateway high latency"
+          description: "Kong latency is {{ $value }}s for service {{ $labels.service }}"
+
+      - alert: KongPluginError
+        expr: rate(kong_plugin_errors_total[5m]) > 1
+        for: 2m
+        labels:
+          severity: warning
+          team: gateway
+        annotations:
+          summary: "Kong plugin errors"
+          description: "Kong plugin {{ $labels.plugin_name }} is generating errors"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)